simulator/config.py

```diff
@@ -1,11 +1,18 @@
-""" Defines various color constants. """
-
-# Cube colors.
-CUBE_RED = "#DB4D67"
-CUBE_BLUE = "#146687"
-CUBE_GOLD = "#87821B"
-
-# Default screen color.
-SCREEN = "#35A6D4"
-# Button color.
-BUTTONS = "#051B24"
+import logger

```

Comment:
I would prefer a config file that was not a Python module. Have a look at [ConfigParser()](https://docs.python.org/2/library/configparser.html) or [PyYaml](https://pyyaml.org/).